interop-cats/jvm/src/main/scala/scalaz/zio/interop/catsjvm.scala

```diff
@@ -27,6 +27,22 @@ import scala.concurrent.duration.{ FiniteDuration, NANOSECONDS, TimeUnit }
 
 abstract class CatsPlatform extends CatsInstances {
   val console = interop.console.cats
+
+  object implicits {
+    implicit def taskTimer(implicit T: effect.Timer[ZIO[Clock, Throwable, ?]]): effect.Timer[Task] =
```

Comment:
For improved performance, you could write this one:

```scala
implicit val taskTimer: effect.Timer[Task] = ???
```

Then over on the right hand side, use the `Clock.Live` methods directly to implement the functionality.

Reply:
If we want it to work for `UIO`, maybe we make it polymorphic in `E`:

```scala
implicit def taskTimer[E]: effect.Timer[IO[E, ?]] = ...
```